database/firebird/firebird.go

```diff
@@ -219,7 +220,7 @@ func (f *Firebird) ensureVersionTable() (err error) {
 
 	query := fmt.Sprintf(`EXECUTE BLOCK AS BEGIN
 			if (not exists(select 1 from rdb$relations where rdb$relation_name = '%v')) then
-			execute statement 'create table "%v" (version bigint not null primary key, dirty boolean not null)';
+			execute statement 'create table "%v" (version bigint not null primary key, dirty smallint not null)';
```

Comment:
Why not use `bit`?
https://firebirdsql.org/manual/migration-mssql-data-types.html

Reply:
Unfortunately Firebird 2.5 doesn't have a BIT type. 
Instead of that it's recommended to use CHAR(1) or SMALLINT
http://www.firebirdfaq.org/faq12/

I prefer number because for me it's more obvious.

If you wish, I can replace it with CHAR(1)